docs/modules/ROOT/nav.adoc

```diff
@@ -295,6 +295,7 @@
 
 .Appendices
 * xref:release-notes/release-notes-history.adoc[History of Release Notes]
+** xref:release-notes/release-notes-2021-1.adoc[Payara Community 5.2021.1 Release Notes]
```

Comment:
No need to duplicate the current release notes page here. Antora doesn't treat duplicate links nicely and would always jump in the sidebar to the page in the Overview even if users click on the link in the History of Release Notes section
```suggestion
```